README.md

```diff
@@ -44,6 +44,22 @@ steps:
   run: ls -R
   working-directory: path/to/artifact
 ```
+
+## Compatibility between `v1` and `v2`
+
+When using `download-artifact@v1`, a directory denoted by the name of the artifact would be created if the `path` input was not provided. All of the contents would be downloaded to this directory.
+```
+   current/working/directory/
+      my-artifact/
+          ... contents of my-artifact
+```
+
+With `v2`, there is no longer an extra directory that is created if the `path` input is not provided. All the contents are downloaded to the current working directory.
```

Comment:
This really isn't about the existence of `path` input though.  In v2, we just do not create the artifact root folder.  You could say something like "to emulate the v1 behavior, use `path` parameter and specific the where you want to download the artifact contents."

An example with `path` input would help too.

Reply:
I added an example of how users can achieve the same behavior as `v1` if they set the `path` input to the name of the artifact.